MeetU/MeetU/Scripts/Directives/datetimepicker.js

```diff
@@ -0,0 +1,21 @@
+﻿(function () {
+    "use strict";
+    angular
+       .module("meetupModule")
+       .directive('datetimepicker', datetimepicker);
+    function datetimepicker() {
+        return function(scope, element, attrs) {
+            element.datetimepicker({
+                inline: false,
+                minDate: '0',
+                dateFormat: 'dd.mm.yy',
```

Comment:
if dateFormat is dd.mm.yy, is timepicker working fine? the datepicker and time picker should work in one selection